lib/bencher_json/src/project/measure/defs.rs

```diff
@@ -0,0 +1,128 @@
+use crate::JsonNewMeasure;
+use bencher_valid::NameId;
+
+pub trait MeasureDefinition {
```

Comment:
Likewise, this should be renamed to `BuiltInMeasure` or whatever we decide.

Reply:
Sure